### PR TITLE
[TASK] Add Daniel and Łukasz as co-maintainers

### PR DESCRIPTION
**Pull request by oliverklee**
Also change Oliver Klee from "developer" to "maintainer" (which better describes the role).

Also change the author company to the Best Practices Team, as this is the team that currently is maintaing the extension, and the TYPO3 Trainer Network has been disbanded.

### CODE REVIEW

**Review by DanielSiepmann — APPROVED** [review]

**Comment by DanielSiepmann** [comment]
Shouldn't we add Łukasz as well?

**Comment by oliverklee** [comment]
> Shouldn't we add Łukasz as well?

Yes, I’d like to. I’ve chatted him up to ask for his okay for this and about which email address to use. (I’d like to move that to a separate PR then.)

**Comment by oliverklee** [comment]
Done.

**Review by DanielSiepmann — APPROVED** [review]

File: ext_emconf.php [hunk 1]
```diff
@@ -17,9 +17,9 @@
     'state' => 'stable',
     'uploadfolder' => false,
     'createDirs' => '',
-    'author' => 'Oliver Klee',
-    'author_email' => 'typo3-coding@oliverklee.de',
-    'author_company' => 'TYPO3 Trainer Network',
+    'author' => 'Oliver Klee, Daniel Siepmann, Łukasz Uznański',
+    'author_email' => 'typo3-coding@oliverklee.de, coding@daniel-siepmann.de',
```
Comment (DanielSiepmann) [inline ext_emconf.php:21]: Łukasz email is missing here.

**Review by DanielSiepmann — CHANGES_REQUESTED** [review]
Missing email in emconf

File: ext_emconf.php (hunk 1 above)
Comment (oliverklee) [reply ext_emconf.php]: Done.

**Review by DanielSiepmann — APPROVED** [review]

**Pull request merged**